packages/metrics/src/Metrics.ts

```diff
@@ -943,6 +972,7 @@ class Metrics extends Utility implements MetricsInterface {
     this.setNamespace(namespace);
     this.setService(serviceName);
     this.setDefaultDimensions(defaultDimensions);
+    this.setDisabled();
```

Comment:
For future usage, I'd move this up after `this.setCustomConfigService(customConfigService);`, so if we want to take in account this value in other setup actions we can do it.